examples/community/ASharedJourney/animation/game_animator.go

```diff
@@ -0,0 +1,75 @@
+//package for managing animations and refreshing screen
+package animation
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/gandrin/ASharedJourney/mechanics"
```

Comment:
`s/github.com\/gandrin/github.com\/faiface\/pixel\/examples\/community\/ASharedJourney/`

Reply:
Yes very true , my bad , let me fix this tonight a

Reply:
Hello , should be all fixed.

Reply:
You might have forgotten to push your changes. Try pushing it to your branch so that it updates this PR?